src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+function Article({ title, date, text }) {
+  return (
+    <>
```

Comment:
Don't use React fragment when not needed. Here - `div` wrapper is better.
```suggestion
    <div>
```